tests/lib/rule-tester/rule-tester.js

```diff
@@ -2555,6 +2580,32 @@ describe("RuleTester", () => {
 
         });
 
+        it("should emit a deprecation warning when CodePath#currentSegments is accessed", () => {
```

Comment:
This test is a duplicate and fails because the deprecation warning for the same rule name has already been emitted from the other test.

Reply:
Sorry about that. Merge conflicts got all crazy on me.